### DISCUSSION TITLE
Piggy Banks: mark that the piggy has already been filled this month

### DISCUSSION THREAD

**Discussion by macuros** [firefly-iii/firefly-iii#5270] [Firefly III]
### Support guidelines

- [X] I've read the [support guidelines](https://github.com/firefly-iii/firefly-iii/blob/main/.github/support.md)
- [X] My request is not listed as [a very good idea, but unfortunately...](https://docs.firefly-iii.org/firefly-iii/about-firefly-iii/what-its-not/)
- [X] I've used [the search](https://github.com/firefly-iii/firefly-iii/issues?q=is%3Aissue) and this has not been discussed before.

### Description

Hello,

I would like to see some kind of mark (maybe with a colour) if the certain piggy bank has already been filled the current month. And in the opposite, if it hasn't been. That way I can be sure that I don't add money to a piggy bank twice in the same month. 

I really like piggy banks and use them a lot. This is a really great feature to set a personal goal. 

### Solution

Maybe some coloured mark in additional column.

**Comment by JC5** [comment]
There is a bar, right? That fills to 100%. Is that not what you were looking for?

**Comment by macuros** [comment]
Yes, but for instance, I have a target amoung of $100 in 5 months. I get a suggestion amoung of $20 every month. So I add that $20 to a piggy bank current month. Then let's say tomorrow I'm not sure if I added those 20 or not. If there will be an indicator that you've already contributed the amount the current month, then it will be easier to track if you have many piggy banks as I do. 

I'm just starting to use FireFly so maybe I'm missing something. I'm a YNAB user trying to replace it with self-hosted finance manager liek FireFly. So, in YNAB there's an indicator if you've already added the monthly suggested amount to the piggy bank. And I find this really useful as it's way easier to track it.

Or maybe I'm missing something. If the bar is orange it's 0, if blue it's somewhere in the middle, if it's green it's 100%. Not sure if there already indicates this, I'm using FireFly only for a few days now.

**Comment by JC5** [comment]
Don't treat them like YNAB's budgets, they're not the same. A piggy bank is a savings goal, for a new car or a PC or whatever. It's not a monthly mandatory budget like YNAB.

If you don't add money to it, well then you don't. There's not going to be an indicator.

**Comment by JC5** [comment]
- https://docs.firefly-iii.org/firefly-iii/advanced-concepts/piggies/
- https://docs.firefly-iii.org/firefly-iii/faq/general/#how-does-it-compare-to-ynab-mint-gnucash-excel

**Comment by macuros** [comment]
Ok, understand. Then I'll try to focus when adding it to piggy banks. I know they are not budgets and I do treat them as goals. That is the main point. In YNAB I actually didn't use the budgeting system, but I've found a workaround that suited to me to use it as goals. And it worked for 4 years to me. 

But not a big deal, I'll try to get use to this way in FireFly.
